spread.yaml

```diff
@@ -230,7 +231,10 @@ prepare: |
     go get ./tests/lib/snapbuild
 
     # Build fakestore.
-    go get ./tests/lib/fakestore/cmd/fakestore
+    if [ "$REMOTE_STORE" = staging ]; then
+        fakestore_tags="-tags withstagingkeys"
+    fi
+    go get $fakestore_tags ./tests/lib/fakestore/cmd/fakestore
```

Comment:
this gives  /bin/bash: line 150: fakestore_tags: unbound variable if REMOTE_STORE is not staging